tui/messages/challenger2_m4_adversarial_test.go

```
@@ -0,0 +1,887 @@
+		}
+	}()
+
+	// Run stress harness for 1.2 seconds under -race
+	time.Sleep(1200 * time.Millisecond)
+	close(stop)
+	wg.Wait()
+}
+
+func makeRandomBytes(n int) []byte {
+	b := make([]byte, n)
+	_, _ = rand.Read(b)
```

golangci-lint (2.13.2)

[error] SA1019: math/rand.Read has been deprecated since Go 1.20 because it shouldn't be used: For almost all use cases, [crypto/rand.Read] is more appropriate. If a deterministic source is required, use [math/rand/v2.ChaCha8.Read]. 

(staticcheck)